certified-connectors/Viafirma/apiDefinition.swagger.json

```diff
@@ -0,0 +1,341 @@
+{
+  "swagger": "2.0",
+    "info": {
+      "version": "1.0.0",
+      "title": "Viafirma",
+      "description": "Send sign requests to others, save your signed documents and much more thanks to Viafirma's connector",
+      "contact": {
+        "name": "Viafirma Support",
+        "url": "https://soporte.viafirma.com/",
+        "email": "helpdesk@viafirma.com"
+      }
+    },
+  "host": "services.viafirma.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/documents/api/v3/messages/": {
+      "post": {
+        "summary": "Send sign request through email",
+        "description": "Create and send a sign request through email",
+        "operationId": "SendSignRequest",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json\r",
+            "description": "Content-Type",
+            "x-ms-summary" : "Content-Type"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "*/*\r",
+            "description": "Accept",
+            "x-ms-summary" : "Accept"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "groupCode": {
+                  "type": "string",
+                  "description": "groupCode",
+                  "title": "",
+                  "x-ms-dynamic-values": {
+                    "operationId": "WhoIAm",
+                    "value-path": "code",
+                    "value-collection": "groupsInfo",
+                    "value-title": "code"
+                  }
+                },
+                "workflow": {
+                  "type": "object",
+                  "properties": {
+                    "type": {
+                      "type": "string",
+                      "description": "type",
+                      "title": "",
```

Comment:
@anfuca Empty property values can be removed until unless you have a specific reason to keep them.